src/engineering/nextauth.mdx

```diff
@@ -0,0 +1,44 @@
+# Next Auth
+
+#### 6th March by Theo Fernandez
+
+[Next Auth](https://next-auth.js.org/) is an authentication library for Next.js applications. It provides a flexible API for authentication, and is designed to work with Next.js and any O-Auth provider.
```

Comment:
why did we need auth in the first place? what benefits did we get from it? why did we implement role based auth?